src/main/resources/application.yaml

```diff
@@ -448,6 +448,13 @@ notification:
   caseworker:
     spec:
       receiver: civilmoneyclaimsdemo@gmail.com
+
+mediation:
+  emails:
+    sender: hmcts.civil@gmail.com
+    recipient: smallclaimsmediation@justice.gov.uk
```

Comment:
this one should be civilmoneyclaimsdemo@gmail.com dont want to send the email to the client when testing locally 😛